pkg/genericcontrolplane/clientutils/multiclusterconfig.go

```diff
@@ -35,8 +37,9 @@ import (
 	"k8s.io/apiserver/pkg/server"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/client-go/rest"
-	_ "k8s.io/component-base/metrics/prometheus/workqueue" // for workqueue metric registration
+	_ "k8s.io/component-base/metrics/prometheus/workqueue"
 	"k8s.io/klog/v2"
+	// for workqueue metric registration
```

Comment:
Does this comment make any sense separated from the import? Also, unrelated formatting change.

Reply:
fixed